packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -34,6 +34,7 @@ export interface RichTextEditorProps
   onChange: (content: EditorContentArray) => void
   value: EditorContentArray
   labelText: ReactNode
+  labelledBy: string
```

Comment:
@lavanya-everest I was hoping for a [discriminated union](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions) here, to ensure that engineers must pass either `labelText` or `labelledBy`.

As this currently stands, engineers will have to pass **both**, which is not quite what we're after. The easy solution is to make both of them optional, but given accessibility implications, I would like to enforce that one of 2 needs to be provided, and TypeScript is the best way to do that.